scanners/subfinder/examples/securecodebox.io/active/scan.yaml

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: "execution.securecodebox.io/v1"
+kind: Scan
+metadata:
+  name: "subfinder-active-securecodebox.io"
+spec:
+  scanType: "subfinder"
+  parameters:
+    - "-active"
+    - "-oI"
```

Comment:
for more "cryptic" parameters in the example:
would be good to have a short explanation what this parameter does, and ideally use the longer form parameter (i think `-ip` in this case. {(onger might not apply here :D)) to reduce confusion